Sources/Swift/Extensions/StringExtensions.swift

```diff
@@ -37,4 +37,20 @@ extension String {
         let end = index(startIndex, offsetBy: range.upperBound)
         return self[startIndex..<end]
     }
+    
+    func snakeToCamelCase() -> String {
+        var result = ""
+        
+        var toUpper = false
+        for char in self {
+            if char == "_" {
+                toUpper = true
+            } else {
+                result.append(toUpper ? char.uppercased() : char.lowercased())
```

Comment:
`m`: I'm not sure if this is a problem, but if you perform this method on `hello_worldHello`, the outcome is `helloWorldhello`, and I think it should be `helloWorldHello` because we only use uppercase if the previous char was _. This logic doesn't keep existing uppercase chars.

Reply:
I dont have a preference. I just followed android implementation,
but I believe the premises is that snake case, [by convention](https://en.wikipedia.org/wiki/Snake_case), dont have upper cases in the middle of the word.

> Snake case (stylized as snake_case) is the naming convention in which each space is replaced with an underscore (_) character, and words are written in lowercase


@romtsn Thoughts?

Reply:
hm, I'm not sure the android implementation is the same, we only uppercase the first character after the underscore, but keep the case of the other characters. So we never lowercase anything

Reply:
Ow, so I got it wrong. Ok, I can change it.